addons/zeus/functions/fnc_ui_defendArea.sqf

```diff
@@ -54,21 +57,28 @@ private _fnc_onUnload = {
 };
 
 private _fnc_onConfirm = {
-    params [["_ctrlButtonOK", controlNull, [controlNull]]];
+    params [["_ctrlButtonOK", controlNull, [controlNull]],["_g_sc"
+		,curatorSelected select 0		//[1]
+	]];
 
     private _display = ctrlParent _ctrlButtonOK;
     if (isNull _display) exitWith {};
 
+	// scrap [1]
     private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
-    if (isNull _logic) exitWith {};
-
+    if (isNull _logic and (_g_sc == [])) exitWith {};
     private _unit = effectiveCommander (attachedTo _logic);
+	if (!isNull _unit) then{ _g_sc = [_unit]; };
+
     private _radius = GETVAR(_display,GVAR(radius),100);
     private _position = getPos _logic;
 
-    [QGVAR(moduleDefendArea), [_unit, _position, _radius], _unit] call CBA_fnc_targetEvent;
+    {
+    	_unit = effectiveCommander _x;
+		[QGVAR(moduleDefendArea), [_unit, _position, _radius], _unit] call CBA_fnc_targetEvent;
+	}forEach _g_sc;
     deleteVehicle _logic;
 };
 
 _display displayAddEventHandler ["Unload", _fnc_onUnload];
-_control ctrlAddEventHandler ["ButtonClick", _fnc_onConfirm];
+_control ctrlAddEventHandler ["ButtonClick", _this + [_g_sc] call _fnc_onConfirm];
```

Comment:
Since in your params you expect `_g_sc` to be part of `_this`, this doesn't make any sense to me.